awx/ui_next/src/components/ActionButtonCell/ActionButtonCell.jsx

```diff
@@ -0,0 +1,8 @@
+import DataListCell from '@components/DataListCell';
+import styled from 'styled-components';
+
+export default styled(DataListCell)`
+  & > :not(:first-child) {
+    margin-left: 20px;
+  }
+`;
```

Comment:
If you stick a `.displayName = 'ActionButtonCell'` on the styled component, it should make it findable in test via `wrapper.find('ActionButtonCell')`. It can also prevent confusion as `.find('DataListCell')` will find both the PF component and the styled component.